lms/djangoapps/support/views/program_enrollments.py

```diff
@@ -297,3 +244,131 @@ def _get_org_key_names(self):
         ).select_related('organization')
 
         return [saml_provider.organization.short_name for saml_provider in saml_providers]
+
+
+class ProgramEnrollmentsInspectorView(ProgramEnrollmentInspector, View):
+    """
+    The view to search and display the program enrollments
+    information of a learner.
+    """
+    exclude_from_schema = True
+    CONSOLE_TEMPLATE_PATH = 'support/program_enrollments_inspector.html'
+
+    @method_decorator(require_support_permission)
+    def get(self, request):
+        """
+        Based on the query string parameters passed through the GET request
+        Search the data store for information about ProgramEnrollment and
+        SSO linkage with the user.
+        """
+        search_error = ''
+        edx_username_or_email = request.GET.get('edx_user', '').strip()
+        org_key = request.GET.get('org_key', '').strip()
+        external_user_key = request.GET.get('external_user_key', '').strip()
```

Comment:
Can we have `None` as default because later we are checking `if`?

Reply:
This is refactored from the old view. This will be deprecated eventually so it doesn't matter. Also, it is an empty string instead of `None` here because of Django Templates I think.